### PR TITLE
Bump libogg and libvorbis, add SDL2 and debugnet

### PR DESCRIPTION
**Pull request by sergiou87**
Just what the title says 😄  While porting https://github.com/sergiou87/open-supaplex to PS3 I had a few issues with sound and graphics, so I started updating/porting things until I got everything working 😂 
- First the music was a bit choppy. I tried updating libogg and libvorbis and didn't fix it, but didn't break anything either, so I thought it'd be good to have them updated.
- Since that didn't help (and updating SDL_mixer didn't help either), I decided to try with SDL2_mixer, which I added in this other PR: https://github.com/zeldin/SDL_PSL1GHT_Libs/pull/3
However, in order to use SDL2_mixer, first I need an SDL2 port, right?
- That's ok because having SDL 1.3 was a bit painful for me. OpenSupaplex had implementations for SDL2 and SDL1.2, but 1.3 is a mix between both which is painful to handle, so porting SDL2 to PS3 was the best option. The port is here: https://github.com/sergiou87/SDL2_PSL1GHT
It's rebased on top of https://github.com/spurious/SDL-mirror 's `master` branch, so it's completely up-to-date.
With this, my game now fully works without changing the implementation. Perfect music, sound, graphics, input… everything 😌 🎉 
- Last but not least, while working on that SDL2 port, I needed some help to debug. In PS Vita I used https://github.com/psxdev/debugnet so I ported it to PS3 (https://github.com/sergiou87/ps3debugnet) and added it to the list of libraries.

### CODE REVIEW

File: scripts/015-sdl_psl1ght_libs.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # sdl_psl1ght_libs.sh by Naomi Peori (naomi@peori.ca)
 
 ## Download the source code.
-wget --no-check-certificate https://github.com/zeldin/SDL_PSL1GHT_Libs/tarball/master -O sdl_psl1ght_libs.tar.gz
+wget --no-check-certificate https://github.com/sergiou87/SDL_PSL1GHT_Libs/tarball/master -O sdl_psl1ght_libs.tar.gz
```
Comment (sergiou87) [inline scripts/015-sdl_psl1ght_libs.sh:5]: This is pointing to my repo, but I guess I'll change it back to yours if you're happy merging https://github.com/zeldin/SDL_PSL1GHT_Libs/pull/3

Comment (miigotu) [reply scripts/015-sdl_psl1ght_libs.sh:5]: I think it is fine for now, but you should make a note to come back and change it if @zeldin can merge that. The PR on his repo seems ok to me but I do not have permissions there.

**Comment by zeldin** [comment]
This PR broke the build.  The `make install` tries to install in `/usr/local/ps3dev/psl1ght/portlibs/ppu`, but this directory does not exist.  The correct path is `/usr/local/ps3dev/portlibs/ppu`.

**Comment by sergiou87** [comment]
@zeldin you mean these two lines are wrong? https://github.com/sergiou87/ps3debugnet/blob/master/libdebugnet/Makefile#L112

**Comment by sergiou87** [comment]
FWIW I used this very same scripts to build a Docker image from which I could build my game and it works flawlessly: https://github.com/sergiou87/open-supaplex/blob/master/ci/Dockerfile-ps3

**Comment by zeldin** [comment]
Yes, they are wrong.  They install in the wrong place.  All ps3libraries scripts install in `$PS3DEV/portlibs/ppu` except `024-debugnet.sh` which installs in `$PSL1GHT/portlibs/ppu`.  I suspect `flipacholas/ps3devextra` contains some old cruft and that is why your Docker has a directory `$PSL1GHT/portlibs/ppu`, but that is not where this should go.  `$PS3DEV/portlibs/ppu` has been the correct place since 2011.  :smile:

**Comment by sergiou87** [comment]
Sorry about that! I'll fix it immediately :sweat:

**Comment by zeldin** [comment]
No worries.  It's on the `develop` branch fortunately.  :smile:

**Comment by sergiou87** [comment]
Should be fixed now with this: https://github.com/sergiou87/ps3debugnet/commit/47c3644af08836109d6845afb2d3eeebdb8807de

**Comment by zeldin** [comment]
Yup, now `024-debugnet.sh` completes successfully for me.  Thanks.

**Pull request merged**